editor/src/components/canvas/controls/new-canvas-controls.tsx

```diff
@@ -596,6 +606,20 @@ const NewCanvasControlsInner = (props: NewCanvasControlsInnerProps) => {
                   isSelectOrInsertMode(editorMode) &&
                     !EP.multiplePathsAllWithTheSameUID(localSelectedViews),
                   <>
+                    {when(
+                      dragInteractionActive,
+                      <CanvasOffsetWrapper>
+                        {gridsWithoutControl.map((grid) => {
```

Comment:
can't these controls be added by the reparent metastrategy? we only want this to happen when it's a drag-to-reparent!